refugee/refugeeController.js

```diff
@@ -5,6 +5,39 @@ var async = require('async');
 var _ = require('lodash');
 var moment = require('moment');
 
+// Create endpoint /api/refugee/:refugee_id for GET
+module.exports.getRefugee = function(req, res) {
+    // Use the Refugee model to find a specific refugee
+    Refugee.findById(req.params.refugee_id, function(err, refugee) {
+        if (err) {
+            res.status(500).send(err)
+            return;
+        };
+
+        res.json(refugee);
+    });
+};
+
+// Create endpoint /api/refugee/:refugee_id for PUT
+module.exports.putRefugee = function(req, res) {
```

Comment:
Here the risk is if you update a document with some data, you have to be sure that you only update fields that you are working with. For instance if `name` is not updated but you still set `name` field as `undefined`, it will overwrite name field. Could you please verify this situation?